src/multibody/parser/urdf-with-geometry.hpp

```diff
@@ -1,98 +0,0 @@
-//
```

Comment:
I really don't know what happened here ( it's the only file that caused problems).
May be just using git move to geometry.hxx won't work if another file already moved to this one ( i'm thinking of urdf-with-geometry.hxx, not hpp)